include/secp256k1.h

```diff
@@ -5,6 +5,83 @@
 extern "C" {
 # endif
 
+/** Opaque data structure that holds context information (precomputed tables etc.).
+ *
+ *  The purpose of context structures is to cache large precomputed data tables
+ *  that are expensive to construct, and also to maintain the randomization data
+ *  for blinding.
+ *
+ *  Do not create a new context object for each operation, as construction is
+ *  far slower than all other API calls (~100 times slower than an ECDSA
+ *  verification), and then use the context for all other operations.
```

Comment:
I think part of this sentence is missing; you say "do not do this .. and then" rather than "do not do this .. instead do this, and then".